addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/CommonRpcParser.java

```diff
@@ -121,4 +125,66 @@ protected Object convertToType(HmDatapoint dp, Object value) {
         }
     }
 
+    /**
+     * Assembles a datapoint with the given parameters.
+     */
+    protected HmDatapoint assembleDatapoint(String name, String unit, String type, String[] options, Object min,
+            Object max, Integer operations, Object defaultValue, HmParamsetType paramsetType, boolean isHmIpDevice)
+            throws IOException {
+        HmDatapoint dp = new HmDatapoint();
+        dp.setName(name);
+        dp.setDescription(name);
+        dp.setUnit(StringUtils.replace(StringUtils.trimToNull(unit), "�", "°"));
```

Comment:
The encoding is OK. For some users, the CCU sends this character instead of the degree symbol. This method with the replace [was already available](https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/GetParamsetDescriptionParser.java#L49), it was only moved to another class